cmd/minikube/cmd/start.go

```diff
@@ -167,7 +169,10 @@ func runStart(cmd *cobra.Command, args []string) {
 
 	validateSpecifiedDriver(existing)
 	validateKubernetesVersion(existing)
+
 	ds, alts, specified := selectDriver(existing)
+	validateBaseImage(cmd.Flag(kicBaseImage), ds.Name)
```

Comment:
since we are in the same file, and we have access to cmd, I think it is better to  use something like this
```
if cmd.Flags().Changed(kicBaseImage) {  
...
}
```

Reply:
Updated